part_1.md

```diff
@@ -0,0 +1,155 @@
+
+There is an augmented circuit that integrates both $F$ and $SNARK.V$, where $SNARK.P$ is responsible for generating the next proof $\Pi_{i+1}$. This proof ensures the integrity of the new state $Z_{i+1}$.
+
+In the final verification step, the SNARK verifier $SNARK.V$ uses the final state $Z_n$, the proof $\Pi_n$, and the verification key $vk$ to check the entire sequence's integrity.
+
+
+**Problem of Naive SNARK-Based IVC**
+If the naive SNARK adopts a general pairing-based approach, this method significantly increases recursion overhead and prolongs the overall proving time. The reason is the internal Verifier's need to repetitively handle pairing and related processes at each step.
+
+[zkPairing](https://0xparc.org/blog/zk-pairing-1) and [circom-pairing](https://github.com/yi-sun/circom-pairing) are the works done by the 0xParc, describing an experimental implementation and benchmarking for verifying pairings in Circom circuit.
+
+
+| Metric                 | Optimal Ate Pairing |
+|------------------------|---------------------------|
+| Constraints            | 11.4M                     |
+| Compilation Time       | 1.9 hrs                   |
+| Witness Generation Time| 1 min                     |
+| Proving Key Size       | 6.5 GB                    |
+| Proof Generation Time  | 52 sec                    |
+| Proof Verification Time| 1 sec                     |
+
+
+From their benchmarks, for example, the Optimal Ate pairing metric cites a Proving time of 52 seconds and a Proof verification time of 1 second. From the perspective of Naive SNARK-Based IVC, this implies that each step will take at least 52 seconds for Proving time and 1 second for Proof verification time. As the number of IVC steps increases to 100 or even 1000, this would lead to substantial time consumption and overhead. Therefore, this approach may not be the most efficient for implementing IVC.
+
+## 2.3 FRI family
+So the next topic is what would be the outcome of adopting FRI commitments instead of  polynomial commitments and pairings in ZKP schemes? Such schemes, utilizing FRI commitments, are known as zkSTARKs or the FRI-Family. zkSTARKs are known for their lack of a trusted setup requirement and their quantum resistance, a characteristic owing to the use of hash functions in Merkle tree commitments.
+
+Another feature is the ease of recursive proofs. For instance, in Plonky2, despite the heavy computational demand due to Merkle Proofs constructed with hash functions, Plonk's custom gates enable the streamlined execution of recursive proofs in poly-log.
+
+Individual proofs are independent, making them suitable for parallelization. Some zkVM projects prefer it for its fast recursive proofs and ease of parallelization.
+
+However, a disadvantage is the expansion in Proof Size attributed to FRI's Merkle Proofs. This enlargement escalates the call data and consequently raising the gas cost for verification on Ethereum, reputedly four times that of Groth16.
```

Comment:
I did some research, but could not figure it out and deleted that sentence to avoid ambiguity, leaving only this sentence.
However, a disadvantage is the expansion in proof size attributed to FRI's Merkle Proofs.